modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/SequenceArea.scala

```diff
@@ -55,36 +55,35 @@ object SequenceStepsTableContainer {
 */
 object SequenceTabContent {
 
-  final case class Props(site: SeqexecSite, p: ModelProxy[InstrumentStatusFocus]) {
+  final case class Props(site: SeqexecSite, p: ModelProxy[InstrumentTabContentFocus]) {
     protected[sequence] val connect = SeqexecCircuit.connect(SeqexecCircuit.statusAndStepReader(p().instrument))
   }
 
   private val component = ScalaComponent.builder[Props]("SequenceTabContent")
     .stateless
     .render_P { p =>
-      val InstrumentStatusFocus(instrument, active, id, _) = p.p()
+      val InstrumentTabContentFocus(instrument, active, sequenceSelected) = p.p()
```

Comment:
This is the actual fix for the bug. The last unused param on `InstrumentStatusFocus` is not used but it can change triggering a re-render